geomstats/geometry/stiefel.py

```diff
@@ -78,6 +78,24 @@ def belongs(self, point, tolerance=TOLERANCE):
         belongs = gs.to_ndarray(belongs, to_ndim=2, axis=1)
         return belongs
 
+    def to_grassmannian(self, point):
```

Comment:
This could be a static method: it does not depend on the object (self) nor the class (cls).

Also, should we have it in stiefel.py, or maybe in matrices.py that would gather all the conversions from matrices types? (I have no strong opinion on it, happy to go with whatever you recommend).

Reply:
I put it here for starters but I'm also not sure it's such a very good place... 
We could end up defining the same method in `grassmannian.py` as: 
```
def from_stiefel(self, point):
    return Stiefel(self.n, self.p).to_grassmann(point)
```

Reply:
I was thinking about 2 options: 

1. Chose to implement only methods of the form `mfdA.to_mfdB` in the `mfdA` class by convention, at least for the code, and optionally later link `mfdB.from_mfdA` to this method

2. Define functions `mfdA_to_mfdB` in a separate `mappings.py` file, imported by the mfdA and the mfdB class files

I don't really know which is better... 1. is simple and allows to mfdA methods seamlessly but 2. has the advantage of grouping all mappings together so that it's easier to keep track of which have been implemented or not...

Reply:
Maybe we could keep track of these mappings in a "feature request" issue anyway

Reply:
@opeltre I like both options, I don't really have a strong opinion for one vs the other! Maybe let's define a new issue and ask for other opinions to choose between the two?

Still, a related issue is: how do we name methods that change coordinates representations from one type to another, for example: poincare ball coordinates to extrinsic coordinates in the hyperbolic space? (@tgeral68). It would be good to have a consistent logic with these as well.